Codeinwp/neve
# Fix: use container full width when featured image is missing

### Summary


Added the case with CSS when the featured image container is missing so the text container can take the full width.

### Will affect visual aspect of the product

YES

### Screenshots 


#### Before

![image](https://github.com/Codeinwp/neve/assets/17597852/de668fa5-c600-4380-a078-0010a352f757)


#### After

![image](https://github.com/Codeinwp/neve/assets/17597852/e8c77080-933f-44e7-8087-14a987196d57)


### Test instructions


- Select List as the Blog layout
- Create some posts and set featured images for some of them
- Hide the thumbnail for the blog posts
- Check how the posts that have a featured image look

## Check before Pull Request is ready:



Closes #4138
